libraries/ui/src/default-config/tailwind.css

```
@@ -113,26 +113,26 @@
 
     /* h1 and h2 use Inter Display */
     .bluedot-h1 {
-        @apply text-color-secondary-text text-size-xl font-bold;
+        @apply text-bluedot-navy text-size-2xl font-semibold leading-tight tracking-tight;
         font-family: InterDisplay, var(--font-sans);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)